ringpop.go

```diff
@@ -783,6 +783,15 @@ func (rp *Ringpop) Labels() (*swim.NodeLabels, error) {
 	return rp.node.Labels(), nil
 }
 
+func (rp *Ringpop) SetIdentity(identity string) error {
```

Comment:
Probably not, but this made it possible to build an application that could set its identity via RPC and validate the change got picked up by the rest of the network.